po/fr.po

```diff
@@ -5773,109 +5781,99 @@ msgid "Ignore the packages by future upgrade commands"
-msgstr "Synchroniser les paquets installés"
+msgstr "Marquer les paquets comme installés automatiquement"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:19
 msgid "Remove and delete all associated configuration and data files"
-msgstr "Supprimer tous les fichiers de configuration associés"
+msgstr "Supprimer tous les fichiers de configuration et de données associés"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:20
-#, fuzzy
 msgid "Reinstall the packages"
 msgstr "Réinstaller les paquets"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:21
-#, fuzzy
 msgid "Remove the packages"
-msgstr "Supprime les paquet"
+msgstr "Supprimer les paquets"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:22
-#, fuzzy
 msgid "Display detailed information about the packages"
-msgstr "Afficher l'état des fichiers extraits"
+msgstr "Afficher les informations détaillés sur les paquets"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:23
 msgid "Consider the packages by future upgrade commands"
 msgstr ""
+"Envisager l’inclusion des paquets lors des futures exécutions de upgrade"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:24
-#, fuzzy
 msgid "Mark packages as manually installed"
-msgstr "Synchroniser les paquets installés"
+msgstr "Marquer les paquets comme installés manuellement"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:25
-#, fuzzy
 msgid "Search for packages matching one of the patterns"
-msgstr "Cherche les paquet contenant le motif"
+msgstr "Chercher les paquets contenant un des motifs"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:26
 msgid "Display brief summary of the available commands and options"
 msgstr "Afficher un sommaire des commandes et options disponibles"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:27
-#, fuzzy
 msgid "Show explanations of automatic installations and removals"
-msgstr "Mentienir un paquet, empêchant suppression ou installation automatique"
+msgstr ""
+"Afficher des explications concernant les (dés)installations automatiques"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:30
-#, fuzzy
 msgid "Purge packages that are not required by any installed package"
-msgstr "Reconstruire et installer un paquet installé"
+msgstr ""
+"Purger les paquets qui ne sont pas des dépendances d’autres paquets installés"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:31
 msgid "Always display a prompt"
-msgstr ""
+msgstr "Toujours afficher une invite"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:32
-#, fuzzy
 msgid "Do not treat recommendations as dependencies"
-msgstr "Corriger les dépendances brisées"
+msgstr "Ne pas traiter les recommandations comme des dépendances"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:33
-#, fuzzy
 msgid "Treat recommendations as dependencies"
-msgstr "Corriger les dépendances brisées"
+msgstr "Ne pas traiter les recommandations comme des dépendances"
```

Comment:
This should actually be the inverse, “Traiter les…” (do), not “Ne pas traiter les…” (do not).